src/stream.ts

```diff
@@ -60,10 +62,24 @@ export class Streamer {
         )
       }
 
+      if (this.reconnectAttempts >= this.maxRetries) {
+        this.retriesExhausted = true
+        if (this.configurations.pollingEnabled) {
+          this.logErrorMessage('Max streaming retries reached. Staying in polling mode.')
+        } else {
+          this.logErrorMessage('Max streaming retries reached.')
```

Comment:
Wondering would it be a good idea to indicate that polling will also be stopped in this `else` statement log so that users would know they're gonna stop receiving updates?

Reply:
Thanks - good catch.

Reply:
Updated